tests/test_ipython_magic.py

```diff
@@ -0,0 +1,29 @@
+import re
+
+from manim import tempconfig
+from manim.utils.ipython_magic import _generate_file_name
+
+
+def test_jupyter_file_naming():
+    """Check the format of file names for jupyter"""
+    scene_name = "SimpleScene"
+    expected_pattern = r"[0-9a-zA-Z@_-]+[@_-]\d\d\d\d-\d\d-\d\d[@_-]\d\d-\d\d-\d\d"
```

Comment:
The scene name must be a valid Python identifier, so this regex here can be sharpened a little:

```suggestion
    expected_pattern = r"[0-9a-zA-Z_]+[@_-]\d\d\d\d-\d\d-\d\d[@_-]\d\d-\d\d-\d\d"
```